app/[locale]/(private)/rest-client/[[...params]]/page.tsx

```diff
@@ -1,14 +1,71 @@
+'use client';
+
+import { ResponseData } from '@types';
 import { Request } from '@components';
 import { Response } from '@components';
+import { useState } from 'react';
+import { handleRequest } from '../../../../actions/request';
+import { toast } from 'sonner';
+import { Client } from '@entities';
+import { constructUrl, decodeBase64 } from '@helpers';
+import { useParams, useSearchParams } from 'next/navigation';
+
+export default function RestClient() {
+  const [responseData, setResponseData] = useState<ResponseData | null>(null);
+
+  const params = useParams();
```

Comment:
```suggestion
    const { params } = useParams();
```
тогда дальше по коду можно будет заменить params.params на params

Reply:
сделано